cadastro.php

```diff
@@ -12,19 +12,23 @@
         <form id="form1" name="form1" method="post" action="acaoCadastro.php">
             <label for="nome">Nome:</label>
             <input name="nome" type="text" id="nome" placeholder="Digite seu nome" required>
+            <label for="cpf">CPF:</label>
+            <input name="cpf" type="number" id="cpf" placeholder="Digite seu CPF" required>
```

Comment:
Entendo que para campo CPF, o type é "text". O type "number" o faria ser tratado como um número para fins de cálculos. Usando type="text", a gente permitiria que o usuário inserisse o CPF no formato desejado (com ou sem pontos e traços). No entanto, entendo a preocupação em deixar "aberta" essa escolha. Eu andei pesquisando aqui e acredito que achei uma solução, em javascript, boa nesse caso:
```suggestion
            <input name="cpf" type="text" id="cpf" placeholder="Digite seu CPF" required>

            <script>
                function formatCPF() {
                    const cpfInput = document.getElementById("cpf");
                    const cpfValue = cpfInput.value.replace(/\D/g, '');
                    const formattedCPF = cpfValue.replace(/(\d{3})(\d{3})(\d{3})(\d{2})/, "$1.$2.$3-$4");
                    cpfInput.value = formattedCPF;
                }
                const cpfInput = document.getElementById("cpf");
                cpfInput.addEventListener("input", formatCPF);
            </script>
```
Nesse cenário agora, permitimos que o campo de CPF seja formatado automaticamente à medida que o usuário digita.